code/create_enigma_terms_json.py

```diff
@@ -46,7 +42,19 @@
 )
 
 
-def fetch_gsheet(gsheet_id: str):
+def load_community_terms_manifest(community_config_dir: Path) -> dict:
+    """Load the community terms manifest JSON file."""
+    with open(community_config_dir / COMMUNITY_TERMS_MANIFEST_FILE) as f:
+        return json.load(f)
+
+
+def lowercase_column_names(df: pd.DataFrame) -> pd.DataFrame:
+    """Convert all column names in the DataFrame to lowercase."""
+    df.columns = [col.lower() for col in df.columns]
+    return df
```

Comment:
probably better to use the pandas builtin `rename` here: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.rename.html - the last example is what you want